src/app/components/experiment-workspace/experiment-workspace.component.html

```diff
@@ -9,41 +9,49 @@ <h1 class="t-page-headline">Experiment Workspace</h1>
+                                </mat-chip-list>
+                                <qhana-markdown class="plugin-description" [markdown]="activePlugin.description"
+                                    [editable]="false">
+                                </qhana-markdown>
+                            </div>
+                        </mat-card-content>
+                    </mat-card>
+                    <mat-card class="card no-padding">
+                        <qhana-plugin-uiframe [url]="frontendUrl" (formDataSubmit)="onPluginUiFormSubmit($event)">
+                        </qhana-plugin-uiframe>
+                    </mat-card>
+                </div>
             </div>
-            <div *ngIf="activePlugin != null">
-                <mat-card class="plugin-details card">
-                    <mat-card-title>
-                        <h2 class="plugin-header" role="button" tabindex="0" aria-controls="plugin-description"
-                            aria-expanded="expandedPluginDescription" (keydown)="onKeyDown($event)"
-                            (click)="expandedPluginDescription = !expandedPluginDescription">
-                            <mat-icon aria-hidden="false" aria-label="expand description"
-                                [hidden]="expandedPluginDescription">chevron_right</mat-icon>
-                            <mat-icon aria-hidden="false" aria-label="expand description"
-                                [hidden]="!expandedPluginDescription">expand_more</mat-icon>
-                            <span class="t-title">{{activePlugin.title ?? activePlugin.identifier}}
-                                (@{{activePlugin.version}})</span>
-                            <qhana-plugin-last-used class="plugin-status" [plugin]="activePlugin" [spinner]="20">
-                            </qhana-plugin-last-used>
-                        </h2>
-                    </mat-card-title>
-                    <mat-card-content id="plugin-description"
-                        class="plugin-description-container {{expandedPluginDescription ? 'open' : 'closed'}}">
-                        <div class="scroll-container">
-                            <mat-chip-list>
-                                <mat-chip *ngFor="let tag of (activePlugin.tags ?? [])">{{tag}}</mat-chip>
-                            </mat-chip-list>
-                            <qhana-markdown class="plugin-description" [markdown]="activePlugin.description"
-                                [editable]="false">
-                            </qhana-markdown>
-                        </div>
-                    </mat-card-content>
-                </mat-card>
-                <mat-card class="card no-padding">
-                    <qhana-plugin-uiframe [url]="frontendUrl" (formDataSubmit)="onPluginUiFormSubmit($event)">
-                    </qhana-plugin-uiframe>
-                </mat-card>
+            <div *ngIf="infoAreaActive">
```

Comment:
Consider creating a new component for this, to avoid the workspace component becoming too bloated.